internal/bus/chainsubscriber.go

```diff
@@ -215,10 +215,12 @@ func (s *chainSubscriber) applyChainUpdate(tx sql.ChainUpdateTx, cau chain.Apply
 
 	// v2 contracts
 	cus = make(map[types.FileContractID]contractUpdate)
-	var createdContracts []types.V2FileContractElement
+	var revisedContracts []types.V2FileContractElement
```

Comment:
`nit` maybe `newContracts` is a better name

Reply:
I thought at first that we just do this for created contracts but actually a contract shows up every time a revision was broadcast and mined. So it really is closer to a list of revised contracts than new contracts.